File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] empty = {};
        if(nums.length < 1){
            return empty;
        }
        
        Map<Integer, Integer> myMap = new HashMap<>();
        
        for(int i = 0; i < nums.length; i++){
            int complement = target - nums[i];
            if(myMap.containsKey(complement)){
                int j = myMap.get(complement);
                int[] results = {j, i};
                return results;
            }
            else{
                myMap.put(nums[i], i);
            }
        }
        return empty;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        num_set = set()
        for num in nums:
            num_set.add(num)
        
        hashmap = {}
        for num in nums:
            if num - 1 not in num_set:
                hashmap[num] = []
                i = 1
                while num + i in num_set:
                    hashmap[num].append(num + i)
                    i += 1
                    
        result = 0
        for value in hashmap.values():
            if len(value) + 1 > result:
                result = len(value) + 1

        return result
